go.mod

```diff
@@ -54,7 +54,7 @@ require (
 	golang.org/x/sys v0.0.0-20200124204421-9fbb57f87de9 // indirect
 	golang.org/x/tools v0.0.0-20191009213438-b090f1f24028
 	google.golang.org/grpc v1.29.1
-	google.golang.org/protobuf v1.24.0
+	google.golang.org/protobuf v1.25.0
```

Comment:
why do we need to upgrade this?

Reply:
Not needed, but happened automatically as I installed the latest version of `protoc-gen-go`. So reverted